Makefile

```diff
@@ -636,41 +636,63 @@ validate-rstudio-image: bin/kubectl
 		continue; \
 	fi; \
 
-# This is only for the workflow action
+# This recipe used mainly from the Pipfile.locks Renewal Action
+# Default Python version
+PYTHON_VERSION ?= 3.11
+ROOT_DIR := $(shell pwd)
+BASE_DIRS := base/c9s-python-$(PYTHON_VERSION) \
+        base/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/minimal/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/datascience/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/pytorch/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/trustyai/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/rocm/tensorflow/ubi9-python-$(PYTHON_VERSION) \
+        jupyter/rocm/pytorch/ubi9-python-$(PYTHON_VERSION) \
+		codeserver/ubi9-python-$(PYTHON_VERSION) \
+		runtimes/minimal/ubi9-python-$(PYTHON_VERSION) \
```

Comment:
indentation intentional?

Reply:
Oh.. I don't know why it is shown like this on the Git preview on vscode looks aligned... :thinking: 
 
 
![image](https://github.com/user-attachments/assets/16dfe841-719e-4b04-afcf-71c7e11a0154)

Reply:
tab vs space probably?

Reply:
This fixed now, but the other `OPT_DIR` still looks shity